build_tools/docker/manage_images.py

```diff
@@ -197,45 +190,59 @@ def update_references(image_url: str, digest: str, dry_run: bool = False):
             )
 
 
-def parse_prod_digests() -> Dict[str, str]:
-    image_urls_to_prod_digests = {}
-    with open(utils.PROD_DIGESTS_PATH, "r") as f:
-        for line in f:
-            image_url, digest = line.strip().split("@")
-            image_urls_to_prod_digests[image_url] = digest
-    return image_urls_to_prod_digests
-
+def main(
+    images: Sequence[str],
+    only_references: bool,
+    docker_dir: pathlib.Path,
+    dry_run: bool,
+):
+    image_deps = json.loads((docker_dir / IMAGE_DEPS_FILENAME).read_text())
+    if "all" in images:
+        images = list(image_deps.keys())
+    for image in images:
+        if image not in image_deps:
+            raise ValueError(
+                f'Image "{image}" not found in the image graph.'
+                f' Available options: all,{",".join(image_deps.keys())}'
```

Comment:
Uber-nit: I think we usually use single quotes for things quoted within error messages, probably because we use double quotes for strings and then we don't have to swap as often. Both of these are basically based on my personal preference and the fact that I wrote most of our Python scripting stuff early on, so now everyone is stuck with those conventions :-P

```suggestion
                f"Image '{image}' not found in the image graph."
                f" Available options: all,{','.join(image_deps.keys())}"
```